email_campaigns/email_to_mentors.txt

```diff
@@ -0,0 +1,34 @@
+Subject: You’re mentoring for the mentored sprint at PyCon US!
+
+{Full name},
+
+Thank you for signing up to mentor for the {project} at the mentored sprint for diverse beginners at PyCon US.
+We’re pleased to welcome you to the sprint.
+
+The mentored sprint will take place on **Saturday, May 4th 2:30 PM - 6:30 PM. Room 25C**
+
+Some useful information:
+
+- The mentored sprint is organized as part of PyCon US. You will need to be registered
+  for PyCon in order to attend the mentored sprint. If you have not registered, please do
+  so as soon as possible, because PyCon is a sell out year after year.
+ 
+- We're excited that numerous projects and mentors have signed up for the sprint! Each
+  project will have the opportunity to pitch for their own project to the group. Please
+  be prepared with a **3 minute pitch**.
+
+- We understand that setting up local environment can be time consuming, and the steps
+  can vary from one project to another. To help the event more successful, we recommend
+  that you include detailed setup instruction in your project's contributing guideline well
+  in advance.
+
+- We also recommend that you prepare a set of beginner friendly issues to be worked on
+  during the sprint.
```

Comment:
We also recommend that you prepare at least 5 beginner-friendly issues to be worked on during the sprint. Here are some ideas to get you started:

- Identify issues that require low-effort involvement (i.e. it will take no more than a couple of hours to complete)
- Get creative: select both coding and non-coding issues to work on the day (i.e. design a logo, add sections to the docs, help create a schematic) this will make your project more accessible and inclusive for the sprinters
- Label the issues accordingly on your issue tracker (i.e. beginner-friendly, good-first issue, whatever style you follow)